core/services/keeper/registry_synchronizer_log_listener.go

```diff
@@ -13,22 +13,22 @@ func (rs *RegistrySynchronizer) JobID() int32 {
 }
 
 func (rs *RegistrySynchronizer) HandleLog(broadcast log.Broadcast) {
-	log := broadcast.DecodedLog()
-	if log == nil || reflect.ValueOf(log).IsNil() {
+	logType := broadcast.DecodedLog()
```

Comment:
Why add `Type` to the name here?

Reply:
@j16r because there is a `log` import  so we have a name collision there

Reply:
@begmaroman I'd suggest to call it e.g. decodedLog, or eventLog - as it's not really a type, but the actual log object

Reply:
@PiotrTrzpil done

Reply:
great!